ArduCopter/version.h

```diff
@@ -6,8 +6,8 @@
 
 #include "ap_version.h"
 
-#define ACEVERSION "v1.0.6"
-#define THISFIRMWARE "ArduCopter V4.0.7 ACE Copter v1.0.6"
+#define ACEVERSION "AVEM-v2.3.1
```

Comment:
I realize this variable isn't used anywhere. I think the intent was that it would pack into the `THISFIRMWARE` string. But this is fine.